src/types/rq/plan.ts

```diff
@@ -0,0 +1,16 @@
+import { TColor } from '..';
+
+interface IPlan {
+  id: number;
+  title: string;
+  description: string;
+  color: TColor;
+  isAllDay: true;
+  type: string;
+  startTime: string;
+  endTime: string;
+  categoryId: number;
+  tags: string[];
+}
```

Comment:
임시방편으로 사용하신 것을 알지만, PR 되는 것이므로 정확한 데이터 정정 부탁드립니다!

```suggestion
export const EPlanType = {
  EVENT: 'event',
  TASK: 'task',
  ALARM: 'alarm',
} as const;
interface IPlan {
  id: number;
  title: string;
  description: string | null;
  color: TColor;
  isAllDay: boolean;
  type: (typeof EPlanType)[keyof typeof EPlanType];
  startTime: string;
  endTime: string | null;
  categoryId: number | null;
  tags: string[];
}
```

Reply:
석훈님이 작성하신 부분에서 `type` 부분을 `Union` 을 사용해서

```ts
export const PLAN_TYPE = ['event', 'task', 'alarm'] as const;

type TPlanType = Union<typeof PLAN_TYPE>;

interface IPlan {
  ...
  type: TPlanType
  ...
}
```

로 하는 것은 어떨까요?

Reply:
PLAN_TYPE에 접근하여 타입을 검사하는 경우가 생긴다면 인덱스 값을 비교하여야 하기 때문에 석훈님이 작성하신 걸로 통일하겠습니다!